CHANGELOG.md

```diff
@@ -5,181 +5,204 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes (😱!!!):
-- Non-empty combinators return `NonEmptyList` ([#102](https://github.com/purescript-contrib/purescript-parsing/pull/102))
+- Non-empty combinators now return `NonEmptyList` (#102)
+- Added support for PureScript 0.14 and dropped support for all previous versions (#103)
```

Comment:
#101 did this. #103 might be a separate thing entirely.

Reply:
Seems #93 is also missing.

Reply:
Missing #97? Not sure if it's breaking.

Reply:
Good catches! I've added #101 and #93. #97 is included in the v5.1.0 release, so I've left it out. I've updated #103; I remember doing it as part of 0.14 changes but they're not specific to the 0.14 release, just a breaking change in a dependency.